implementation/contracts/system/VendingMachine.sol

```diff
@@ -9,6 +9,11 @@ import {DepositUtils} from "../deposit/DepositUtils.sol";
 import "../deposit/Deposit.sol";
 import "./TBTCSystemAuthority.sol";
 
+/// @title  Vending Machine
+/// @notice The Vending Machine swaps TDTs (`TBTCDepositToken`)
```

Comment:
I actually prefer to express this in terms of responsibilities, fwiw, but I'm fine with where we landed.